pkg/helper/versions/helpers_test.go

```diff
@@ -65,16 +65,16 @@ var _ = Describe("Version Helpers", Ordered, func() {
 				Expect(minimalVersion).To(Equal(expected))
 			},
 			Entry("Future control plane",
+				"4.17.0",
 				"4.15.0",
-				"4.13.0",
 			),
 			Entry("Nightly control plane",
-				"4.14.0-0.nightly-2023-02-27-084419",
-				"4.12.0",
+				"4.16.0-0.nightly-2023-02-27-084419",
+				"4.14.0",
 			),
 			Entry("Current control plane",
-				"4.12.5",
-				"4.12.0-0.a",
+				"4.14.5",
+				"4.14.0",
```

Comment:
Side-comment for the maintainers, this seems like a source of infinite toil as the minimum supported version keeps creeping up in the service. Perhaps possible to factor this some other way so the tests remain valid but the work is not necessary on every bump.